resultbrowser/src/js/resultbrowser.js

```diff
@@ -1,88 +1,176 @@
-                js  : true,
-                css : true,
-                html: true
+(function () {
+
+    let transaction = null,
+        actionlist = null,
+        requestContent = null,
+        requestText = null,
+        actionContent = null,
+        errorContent = null,
+        postRequestParam = null,
+        requestBodySmall = null,
+        localTimeZone = null,
+        extras = {
+            highlight: false,
+            beautify: {
+                js: false,
+                css: false,
+                html: false
             }
         },
-        $menu = null,
-        $menuIcon = null;
+        menu = null,
+        menuIcon = null,
+        transactionContent = null,
+        valueLog = null;
 
-        $transactionContent = null,
-        $valueLog = null;
 
-    function init() {
+    function ajax(url, options) {
+        return fetch(url, options || {})
+    }
 
-        function cachedScript(url, options){
-            options = $.extend(options || {}, {
-                dataType: 'script',
-                cache: true,
-                url: url
-            });
+    function show(element) {
+        if (element) {
+            element.classList.remove("hide");
+        }
+    }
 
-            return $.ajax(options);
+    function hide(element) {
+        if (element) {
+            element.classList.add("hide");
         }
+    }
 
-        $navigation = $('#navigation');
-        $transaction = $('#transaction');
-        $actionlist = $('#actionlist');
-        $header = $('#header');
-        $leftSideMenu = $('#leftSideMenu');
+    function toggle(element) {
+        if (element) {
+            element.classList.toggle("hide")
+        }
+    }
 
-        $content = $('#content');
+    function getParents(element) {
+        let result = [];
+        for (let parent = element && element.parentElement; parent; parent = parent.parentElement) {
+            result.push(parent);
+        }
+        return result;
+    }
 
-        $requestContent = $('#requestcontent');
-        $requestText = $('#requesttext');
-        $actionContent = $('#actioncontent');
-        $errorContent = $('#errorcontent');
-        $postRequestParam = $('#postrequestparameters');
-        $requestBodySmall = $('#requestBodySmall');
+    function empty(element) {
+        if (element) {
+            while (element.firstChild) {
+                element.removeChild(element.lastChild);
+            }
+        }
+    }
 
-        $transactionContent = $('#transactionContent');
-        $valueLog = $('#valueLog');
+    function getText(element) {
+        if (element) {
+            return element.innerText;
+        }
+    }
 
-        $window = $(window);
+    function setText(element, text) {
+        if (element) {
+            element.innerText = text;
+        }
+    }
 
-        $menu = $('#menu');
-        $menuIcon = $('#menu-icon');
+    function getElementById(id) {
+        return document.getElementById(id);
+    }
```

Comment:
I tried it this way, but an error occured: "<method> called on an object that does not implement interface Document".
This could be probably because because the functions rely on `this` in the document context (see: https://github.com/akhodakivskiy/VimFx/issues/841#issuecomment-263792772).
I have now used an arrow function (const getElementById = id => document.getElementById(id)), this way it works.